non-atomic-action-interpreter-module/README.md

```diff
@@ -0,0 +1,122 @@
+# Non-Atomic Action Interpreter Module
+
+This module is part of the ostis-ps-lib and is responsible for interpreting non-atomic actions within the OSTIS.
+
```

Comment:
The main flow of using components is components managers.  Need to add info about it. Even with the future mark

Reply:
Right now we don't have the ability to put components into tasks using the component manager, so I can't write about it here as it would be misinformation. We'll add this when it's actually true